src/scss/yellow/components/_header-banner.scss

```diff
@@ -165,12 +165,14 @@ $box-shadow: rgba($color-interface-content-extra-dark, .04);
     }
 
     @media screen and (min-width: $media-medium) {
+      box-shadow: none;
```

Comment:
Will we have no shadow if it was not mobile?

Reply:
Yes, it'll have the shadow on desktop! 
The thing is, when I set the box-shadow: none, at desktop (media-normal), the lateral shadow (marked with an arrow on screenshot ) disappear, but the shadow we need, remains! 

<img width="749" alt="Screen Shot 2019-09-26 at 17 15 06" src="https://user-images.githubusercontent.com/42356402/65722054-d0344400-e081-11e9-9535-250614e195b3.png">

Reply:
@gisantin but this is happening because the shadow would have to be in .header-banner__content because it would need to do `none`

Reply:
In fact, no! When I put the box-shadow at header-banner__content, it disappear on mobile and gets messy at desktop (see the screenshot!)

Mobile:
<img width="749" alt="Screen Shot 2019-09-26 at 17 15 06" src="https://user-images.githubusercontent.com/42356402/65724790-89e1e380-e087-11e9-8878-1db24780f7d8.png">

Desktop:
<img width="1033" alt="Screen Shot 2019-09-26 at 17 56 45" src="https://user-images.githubusercontent.com/42356402/65724825-95cda580-e087-11e9-8485-22933b16fd61.png">

Reply:
Sorry! The `box-shadow`has to be added in `.header-banner`